design-guidelines/interactive-element-states.md

```diff
@@ -11,7 +11,33 @@ States communicate the different statuses of UI components to the user, includin
 - Data inputs
   - For example: a text input
 
-## States
+## States: Ready to receive input
+
+### Hover
+
+Styling a component when it is hovered by a cursor indicates that a component can be interacted with. If a component is normally interactive but is currently disabled, it should not be styled on hover. Do not remove the browser's default hover styles unless you're overriding them with custom hover styles.
+
+In addition to updating a component's style when it is hovered, it is also a good idea to update the cursor style to match the purpose of the component. For example: showing a resize cursor when hovering a drag handle that is used to resize a table column.
+
+### Focus
+
+Styling a component when it receives focus indicates that a component is active and ready to receive input. If a component is normally interactive but is currently disabled, it should not receive focus. Do not remove the browser's default focus styles unless you're overriding them with custom focus styles.
+
+Focus styles can match the hover styles, but it's also okay to have different styles to indicate hover and focus. It's also acceptable to have different focus styles for different input devices. For example: our button components have focus styles that only appear when the button is focused using the keyboard.
+
+### Active
+
+A component can be styled to indicate it is actively receiving input such as a mouse click. If a component is normally interactive but is currently disabled, it should not receive active styles. Do not remove the browser's default active styles unless you're overriding them with custom active styles.
```

Comment:
That's why I said "_such as_ a mouse click".

I can change it if the current text is unclear or confusing, but this seems a little wordy 🤔 :
> A component can be styled to indicate it is actively receiving input such as a mouse click **or pressing the Tab key to focus the input**.

What do you think?

Reply:
I think it is worth it because keyboard nav is such a great feature you have included and such an efficiency gain for our users and use cases that I think it is important to emphasize and evangelize that we want everyone showing off and trying out our apps as much as possible with the keyboard.

Reply:
thank you kindly for taking the pass.